profiling/src/profile/internal/label.rs

```diff
@@ -0,0 +1,77 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2021-Present Datadog, Inc.
+
+use super::super::{pprof, Id, StringId};
+
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Hash, PartialOrd, Ord)]
+pub enum LabelValue {
+    Str(StringId),
+    Num {
+        num: i64,
+        num_unit: Option<StringId>,
+    },
+}
+
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Hash, PartialOrd, Ord)]
+pub struct Label {
+    key: StringId,
+    value: LabelValue,
+}
+
+impl Label {
+    pub fn has_num_value(&self) -> bool {
```

Comment:
This could also be `get_num_value(&self) -> Option<(i64, Option<StringId>>`, to allow easy destructuring